File: SEIST/polyglot/clojure/seis_release_contract.clj
(ns seis-release-contract)

(def package-path "dist/seis-static.zip")
(def supported-locales #{"tr" "en" "fr" "it" "de" "es" "ar"})
(def reduced-motion-required true)
(def live-upload-requires-target true)

(defn supports-locale? [locale]
  (contains? supported-locales locale))

File: SEIST/polyglot/haskell/SeisReleaseContract.hs
module SeisReleaseContract where

packagePath :: String
packagePath = "dist/seis-static.zip"

supportedLocales :: [String]
supportedLocales = ["tr", "en", "fr", "it", "de", "es", "ar"]

reducedMotionRequired :: Bool
reducedMotionRequired = True

liveUploadRequiresTarget :: Bool
liveUploadRequiresTarget = True
